kubernetes/samples/scripts/terraform/oke.create.sh

```diff
@@ -32,11 +32,11 @@ setupTerraform() {
     mkdir ${terraformDir}
     cd ${terraformDir}
     if [[ "${OSTYPE}" == "darwin"* ]]; then
-      curl -O https://releases.hashicorp.com/terraform/0.11.15/terraform_0.12.31_darwin_amd64.zip
```

Comment:
Why not include ${OSTYPE} in the URL of the file you are downloading rather than use an if statement. Also, if you used curl -Lo <local-file-name> <remote-url> then the unzip line could always be the same.